### PR TITLE
GameDB: Beeg fixing about everything

### PR DESCRIPTION
**Pull request by RedDevilus**
### Description of Changes

- Before HW Parappa 2:
PaRappa the Rapper 2_SCUS-97167_20230411051710

- After HW Parappa 2:
PaRappa the Rapper 2_SCUS-97167_20230411051719

- SW Parappa 2:
PaRappa the Rapper 2_SCUS-97167_20230411051721

Before HW Socom US Navy Seals Combined Assault:
SOCOM - U S  Navy SEALs - Combined Assault_SCUS-97545_20230329225144

After HW Socom US Navy Seals Combined Assault:
SOCOM - U S  Navy SEALs - Combined Assault_SCUS-97545_20230329224734

Before HW Socom US Navy Seals Combined Assault:
SOCOM - U S  Navy SEALs - Combined Assault_SCUS-97545_20230330011832

After HW Socom US Navy Seals Combined Assault:
SOCOM - U S  Navy SEALs - Combined Assault_SCUS-97545_20230329225449

Before HW Socom 3 Navy Seals:
SOCOM 3 - U S  Navy SEALs_SCES-53300_20230329230304

After HW Socom 3 Navy Seals:
SOCOM 3 - U S  Navy SEALs_SCES-53300_20230329230252

Before HW SSX 3 (recommended blending so only OSD notification):
SSX 3_SLES-51697_20230329142358
After HW SSX 3 (recommended blending so only OSD notification):
SSX 3_SLES-51697_20230329142419

Before HW SSX On Tour (recommended blending so only OSD notification):
SSX On Tour_SLES-53552_20230402013044
After HW SSX On Tour (recommended blending so only OSD notification):
SSX On Tour_SLES-53552_20230402013046

Before HW SSX Tricky (recommended blending so only OSD notification):
SSX Tricky_SLES-50545_20230329143213
After HW SSX Tricky (recommended blending so only OSD notification):
SSX Tricky_SLES-50545_20230329143332

Before HW Blood Will Tell (Dororo) Basic Blend:
Blood Will Tell_SLUS-20782_20230329141553
After HW Blood Will Tell (Dororo) Medium Blend:
Blood Will Tell_SLUS-20782_20230329141700

Before HW Tarzan:
Disney's Tarzan Untamed_SLUS-20076_20230313030945
After HW Tarzan:
Disney's Tarzan Untamed_SLUS-20076_20230313030948

Before HW Winnie the Pooh Rumbly Tumbly Adventure:
Disney's Winnie the Pooh Rumbly Tumbly Adventure_SLES-52889_20230726205503
After HW Winnie the Pooh Rumbly Tumbly Adventure with autoflush:
Disney's Winnie the Pooh Rumbly Tumbly Adventure_SLES-52889_20230726205507

Before HW The Dukes of Hazzard, Return of the General Lee:
Dukes of Hazzard, The - Return of the General Lee_SLUS-20959_20230726223703
After HW The Dukes of Hazzard, Return of the General Lee w normal halfpixel offset:
Dukes of Hazzard, The - Return of the General Lee_SLUS-20959_20230726224206

TODO:
One Piece - Grand Adventure_SLES-54165_20230325231716
### Rationale behind Changes

Less tinkering for users, keep in mind that for example SSX is too heavy on barriers that would not only reduce performance for NVIDIA users but also AMD and Intel users.
### Suggested Testing Steps

Test relevant games.

### CODE REVIEW

**Comment by RedDevilus** [comment]
Need to fix the PR, because I unstashed my old stuff before rebase :(

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -787,6 +787,8 @@ SCAJ-20068:
     - SoftwareRendererFMVHack # Fixes brightness and overlapping subtitles.
   gsHWFixes:
     roundSprite: 1 # Fixes font artifacts.
+    skipDrawStart: 1 # Removes blur in background in battles.
+    skipDrawEnd: 1 # Removes blur in background in battles.
```
Comment (lightningterror) [inline bin/resources/GameIndex.yaml:791]: Skipdraw should be removed, unless absolutely necessary with tc issues.
Same for the others.

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -39049,6 +39199,7 @@ SLPS-25052:
     alignSprite: 1 # Fixes vertical lines.
     mergeSprite: 1 # Fixes vertical lines.
     gpuTargetCLUT: 1 # Fixes broken sun.
+    gpuPaletteConversion: 2 # Stops excessive VRAM usage with preloading on.
```
Comment (Mrlinkwii) [inline bin/resources/GameIndex.yaml:39202]: this isnt needed  also

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -16140,6 +16205,13 @@ SLES-52573:
   region: "PAL-M5"
   roundModes:
     eeRoundMode: 2 # Fixes abnormal character behaviour.
+SLED-52574:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:16208]: why did you move this? it was in the right place, it's SLED not SLES

Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:16208]: Old local file

File: bin/resources/GameIndex.yaml (hunk 1 above)
Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:791]: It's not breaking other effects, I tested it extensively.

File: bin/resources/GameIndex.yaml (hunk 2 above)
Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:39202]: I can remove it but FMVs need to have a fix in the future then, it's not system critical so guess it will be low priority as they don't need that much power

File: bin/resources/GameIndex.yaml (hunk 1 above)
Comment (Mrlinkwii) [reply bin/resources/GameIndex.yaml:791]: The blur is intended , people can make a per game setting if someone hates it

Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:791]: GameIndex with skipdraw:

    skipDrawStart: 1 # Removes blur in background in battles.
    skipDrawEnd: 1 # Removes blur in background in battles.
[GameIndex.zip](https://github.com/PCSX2/pcsx2/files/12178413/GameIndex.zip)

I'll remove it for now, photos are still in a mess but I can fix that later even if PR is merged.

**Comment by RedDevilus** [comment]
PR should be fine now.
